python/cuml/fil/fil.pyx

```diff
@@ -575,14 +575,11 @@ class ForestInference(Base):
             model passed.
 
         """
-        if(has_treelite() is False):
-            raise ImportError("Treelite needs to be built from source"
-                              " for this function to be used.")
+        if(not has_treelite()):
+            raise ImportError("Treelite version 0.9 needs to be installed.")
         else:
-            if(treelite.__version__ <= '0.32'):
-                raise ImportError("Treelite version greater than 0.32 is"
-                                  " required to use this function. Please"
-                                  " build Treelite from source.")
+            if(treelite.__version__ < '0.9'):
```

Comment:
For version comparison, please look at the approach in import_utils (https://github.com/rapidsai/cuml/blob/branch-0.14/python/cuml/common/import_utils.py). It uses looseversion currently and does this in a structured way. Doing string comparisons is not safe. Eg. Treelite 0.10 would fail this.

Reply:
I updated it but wanted to mention this :
```
>>> (LooseVersion(str(treelite.__version__)) > LooseVersion('0.90'))
False
>>> (LooseVersion(str(treelite.__version__)) > LooseVersion('0.9'))
True
```
here for testing purpose the `treelite.__version__ ` used was 0.32

Reply:
The issue is that Treelite isn't yet using semantic versioning. For now, the next Treelite release should be 0.90 to make the version comparison work.